app/views/users/edit.html.erb

```diff
@@ -3,15 +3,19 @@
   <h2><%= t('users.edit.edit_profile') %></h2>
 
   <hr />
+  <% if Rails.env == "production"%>
+  <div class='col-lg-8 col-md-10 col-sm-10' style="display:flex; justify-content: flex-start; padding: 0;">
+  <label> Link your PublicLab.org account to other providers</label>
```

Comment:
Can you make it short like
``Link your Public Lab account to providers``
We usually prefer short heading. Provide the last screenshot and then we are ready to go

Reply:
Also commit your changes before you publish a screenshot.

Reply:
![screenshot from 2018-07-11 00-03-27](https://user-images.githubusercontent.com/32437794/42530088-e3cbcbb4-849d-11e8-89cc-5bd4e545d137.png)

Reply:
@SidharthBansal done